examples/fa_hierarchy/in/fa.asciipb

```diff
@@ -0,0 +1,18 @@
+fixed_values {
+	    #Automatic initialization if size is 0. Use the syntax below to set hyperparameters manually.
+  mutilde: {
+    size:0
+    data:[]
+    #size:20
+    #data:[50.71368, 51.54430, 52.06730, 52.86145, 53.39870, 54.14019, 54.92412, 55.46906, 56.30326, 57.10029, 57.64240, 58.27382, 58.84905, 59.47209, 60.35402, 60.87941, 61.67874, 62.22960, 63.06052, 63.57241]
+    }
```

Comment:
Now we have one file with commented lines if the user wants to manually set parameters as @brunoguindani suggested